beacon-chain/operations/attestations/service.go

```diff
@@ -34,12 +38,18 @@ func NewService(ctx context.Context, cfg *Config) (*Service, error) {
 		return nil, err
 	}
 
+	pruneInterval := cfg.pruneInterval
+	if pruneInterval == 0 {
```

Comment:
I think we can use a comment here for clarity? it's always default to 1s

Readers may not read `prune_expired.go` first

Reply:
Agreed! Code is updated.